### ISSUE TITLE
Allow ctrl-D to detach client from debugger

### ISSUE THREAD

**Issue by kmaork** [kmaork/madbg#10]

**Comment by 3ap** [comment]
```
Waiting for connection from debugger console on 127.0.0.1:3513
Traceback (most recent call last):
  File "./test2.py", line 6, in <module>
    time.sleep(100)
  File "/home/time/madbg/madbg/debugger.py", line 57, in <lambda>
    td = lambda *args: self.trace_dispatch(*args, done_callback=done_callback)
  File "/home/time/madbg/madbg/debugger.py", line 47, in trace_dispatch
    return super().trace_dispatch(frame, event, arg)
  File "/usr/lib/python3.7/bdb.py", line 94, in trace_dispatch
    return self.dispatch_exception(frame, arg)
  File "/usr/lib/python3.7/bdb.py", line 174, in dispatch_exception
    if self.quitting: raise BdbQuit
```

Immediately after I pressed ^D in `madbg`, original process exited with a stack trace. Is this behavior expected?
At least that's not I was expected after reading "Affects the debugged program minimally" :)

**Comment by kmaork** [comment]
Thanks, fixed and released 1.1.3, enjoy 😄

**Comment by mw66** [comment]
@kmaork: can we add a direct `detach` command? Otherwise, this feature is hidden, i.e. in `help` menu, it is not showing. Thanks for the great library!
